### PR TITLE
Removed phone object and resolved conflicts.

### PR DESCRIPTION
**Pull request by ailanthustng**

### CODE REVIEW

File: src/main/resources/view/PersonListCard.fxml [hunk 1]
```diff
@@ -28,7 +28,6 @@
         <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
       </HBox>
       <FlowPane fx:id="tags" />
-      <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
```
Comment (WillySeahh) [inline src/main/resources/view/PersonListCard.fxml:31]: Good job for removing this so it wont show up in the GUI

File: src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json [hunk 2]
```diff
@@ -1,6 +1,5 @@
 {
   "persons": [ {
-    "name": "Hans Muster",
-    "phone": "948dfa2424"
+    "name": "Ha!ns Mu@ster"
```
Comment (WillySeahh) [inline src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json:3]: I understand that this is to cater to some Tests that are expected to fail due to invalidPerson right?

File: src/main/resources/view/PersonListCard.fxml (hunk 1 above)
Comment (ailanthustng) [reply src/main/resources/view/PersonListCard.fxml:31]: Thank you Willy.

File: src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json (hunk 2 above)
Comment (ailanthustng) [reply src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json:3]: That is correct.

**Review by sheexiong — APPROVED** [review]
LGTM

**Pull request merged**